cirq-core/cirq/circuits/qasm_output.py

```diff
@@ -203,6 +203,7 @@ def __init__(
             qubit_id_map=qubit_id_map,
             meas_key_id_map=meas_key_id_map,
         )
+        self.cregs = self._generate_cregs()
```

Comment:
My suggestion from the earlier comment was to assign it to a field in `self.args` in the line above, since `QasmArgs` does get passed around. But that could be done as a separate PR for https://github.com/quantumlib/Cirq/issues/5691. This lgtm for https://github.com/quantumlib/Cirq/issues/6508.

Reply:
Oh, I see.  That makes more sense.  I will look into this in a follow-up PR.